postgres/README.md

```diff
@@ -189,9 +189,17 @@ primary key columns to be `NOT NULL`. However, in PostgreSQL, a primary key is a
 combination of `NOT NULL` and `UNIQUE`, and so primary key columns from
 PostgreSQL will be mapped to Spanner columns that are both primary keys and `NOT NULL`.
 
-### Foreign Keys and Default Values
+### Foreign Keys
 
-Spanner does not currently support foreign keys or default values. We drop these
+The tool maps PostgreSQL foreign key constraints into Spanner foreign key constraints, and
+preserves constraints names where possible. Note that Spanner requires foreign key
```

Comment:
My bad: that should be "constraint names"

Reply:
updated